docs/getting-started-guides/windows/index.md

```diff
@@ -1,149 +1,256 @@
 ---
-title: Windows Server Containers
+title: Using Windows Server Containers in Kubernetes
 ---
-**Note:** These instructions were recently updated based on Windows Server platform enhancements
+**Note:** These instructions were recently updated based on Windows Server platform enhancements and the Kubernetes v1.9 release
 
-Kubernetes version 1.5 introduced support for Windows Server Containers based on the Windows Server 2016 OS. With the release of Windows Server version 1709 and using Kubernetes v1.8 users are able to deploy a K8s cluster either on-premises or in a private/public cloud using a number of different network toplogies and CNI plugins. Platform improvements include:
-- Improved support for Pods! Shared network namespace (compartment) with multiple Windows Server containers (shared kernel)
-- Reduced network complexity by using a single network endpoint per Pod
+Kubernetes version 1.5 introduced Alpha support for Windows Server Containers based on the Windows Server 2016 operating system. With the release of Windows Server version 1709 and using Kubernetes v1.9 users are able to deploy a Kubernetes cluster either on-premises or in a private/public cloud using a number of different network topologies and CNI plugins. Some key feature improvements for Windows Server Containers on Kubernetes include:
+- Improved support for pods! Shared network namespace (compartment) with multiple Windows Server containers (shared kernel)
+- Reduced network complexity by using a single network endpoint per pod
 - Kernel-Based load-balancing using the Virtual Filtering Platform (VFP) Hyper-v Switch Extension (analogous to Linux iptables)
+- Container Runtime Interface (CRI) pod and node level statistics
+- Support for kubeadm commands to add Windows Server nodes to a Kubernetes environment
 
- The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server version 1709.
+The Kubernetes control plane (API Server, Scheduler, Controller Manager, etc) continue to run on Linux, while the kubelet and kube-proxy can be run on Windows Server 2016 or later
 
-**Note:** Windows Server Containers on Kubernetes is an Alpha feature in Kubernetes 1.8.
+**Note:** Windows Server Containers on Kubernetes is a Beta feature in Kubernetes v1.9
+
+## Build
+We recommend using the release binaries that can be found at [https://github.com/kubernetes/kubernetes/releases](https://github.com/kubernetes/kubernetes/releases).
```

Comment:
From Microsoft perspective, we would like to encourage people to create a Linux cluster and then add Windows nodes to the existing cluster. Perhaps this isn't necessary now that it sounds as though kubeadm integration works with Windows as well. Question: will kubeadm work with any CNI plugin on Windows?

Reply:
Usually when people come here it means they already have a Linux cluster and they have a Windows node. I don't think it's even possible to have the all the kubernetes services running on Windows. @bsteciuk can you answer about the question for kubeadm? Thanks!

Reply:
I will let @bsteciuk  chime in as well, but @JMesser81 , kubeadm will add the node to your cluster and then you have to explicitly run the commands to set up the CNI

Reply:
With kubeadm, the steps would be 
1) Initialize the master with kubeadm (Linux)
2) Setup CNI based networking (outside of kubeadm)
3) Join worker nodes to cluster with kubeadm (Linux or Windows)

Reply:
@bsteciuk  this is great info. i think we should have it here in the docs as well. cna you please update the PR with this as well?